lochness/xnat/__init__.py

```diff
@@ -173,13 +176,24 @@ def download_xnat_session_dataorc(
         subject (str): The ID of the subject.
         session (str): The ID of the session to download.
         out_dir (str): The directory to which the session data should be downloaded.
+        xnat_username (str): The username to use when authenticating with the XNAT server.
+        xnat_password (str): The password to use when authenticating with the XNAT server.
 
     Returns:
         None
     """
 
+    dataorc_binary_path = Path('/opt/software/dataorc-bin/dataorc')
+
+    def clear_stored_credentials(dataorc_bindary_path: Path):
```

Comment:
Is this a function within a class?

Reply:
I expected the function on its own will never be called, and had it nested within a function. Should I pull it up?

Reply:
You can decide the best course here.